CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+Thanks for thinking about contributing to Chamber!
+
+Chamber is an open-source project run with ❤️ by Segment. We've made it open source in the hope that other folks will find it useful. That said, making open source software takes a lot of work, so we try to keep Chamber focused on its goals. That means first and foremost supporting the use cases we have at Segment, but any other reasonable additions will be accepted with gratitude.
+
+The purpose of these guidelines is all about setting expectations.
+
+# Feature requests (`enhancement` label)
+
+New features should be requested via Issue first, to decide whether it falls within Chamber's scope. *Don't* start with a feature PR without discussion.
+
+Even if it is decided that a feature fits Chamber's goals, that doesn't imply that someone is working on it. The only people who are obliged to work on a feature are the people who intend to use it. An `enhancement` issue without an assignee or a milestone means that nobody intends to work on it. If you're interested in working on it, just say so and we can assign it to you.
+
+An `enhancement` issue with a milestone means we intend to write it, but haven't decided who will do it yet.
+
+`enhancement` issues are subject to our [Staleness Policy](#Staleness Policy). An `enhancement` that's gone stale means that no one's intending to work on it, which implies the feature isn't really that important. If this isn't the case, commenting during the staleness grace period will freshen it; this should almost always be a commitment to implementing it.
+
+# Timeliness
+
+As a user, there's nothing worse than crafting a beautiful PR with extensive tests only to be met with tumbleweeds from a long-abandoned project. We want to assure you that Chamber is maintained and actively worked on, but give you some guidelines on how long you might expect to wait before things get done.
+
+Issues should be triaged within 1 week, where triaging generally means figuring out which type of issue it is and adding labels.
+
+Pull requests (that have had design approval in an issue) should expect responses within 3 days.
+
+If you're finding we aren't abiding by these timelines, feel free to @-mention someone in [CODEOWNERS](.github/CODEOWNERS) to get our attention. If you're the shy type, don't worry that you're bothering us; you're helping us stick the commitments we've made :)
+
+# Staleness
+
+All issues and PRs are subject to staleness after some period of inactivity. An issue/PR going stale indicates there isn't enough interest in getting it resolved. After some grace period, a stale issue/PR will be closed.
```

Comment:
I think this makes more sense with pull requests than issues, if there is no actionable item or the submitter does not respond then I agree closing it makes sense. I would like to advocate for keeping valid (enhancement) issues with sufficient information open as this means that it can be found by someone looking to make a contribution and walked over the line. Maybe using some additional status and priority labels can help give us the best of both worlds.

Reply:
I think this boils down to the fact that Chamber probably isn't big or popular enough to have people roaming by and looking for features to implement because they're bored. I'd guess that almost every feature is added by someone who wants it personally, or for users at their company.

What I *have* seen happen with old feature requests is users stopping by to ask when it's going to be done. I just would rather have it be made clear that feature requests need people to implement them, or at least care for them.

One thing to note is that anybody in the issue can post anything during the staleness period to make it fresh for another 60 days, probably even an emoji. That's a pretty low bar.

Reply:
Just going to quote my response from https://github.com/segmentio/chamber/pull/191#issuecomment-472131451

TLDR: duplicates are fine; re-opening when interest renews is fine.

> I think I'd rather have to deal with the slight burden of reconciling duplicate feature requests than the bigger burden of a long list of feature requests that will never get done. An open feature request means that there's some reasonable expectation that someone will pick it up and get it done. If we don't expire feature requests, maybe a user will be able to find an existing feature request easier, but it might give them a false sense of hope that it's in progress. A new issue indicates renewed interest, and perhaps a new contributor :)